src/pss.jl

```diff
@@ -107,30 +110,34 @@ Construct SiggeSimulator instance based on simulation
 function SiggenSimulator(simulation_settings::PropDict)
     # set defaults
     inputs = Dict{String, Any}()
-    for k in (:fieldgen_config, :drift_vel, :impurity_profile)
+    for k in (:fieldgen_config, :drift_vel, :impurity_profile, :crystal_metadata_path)
         inputs[string(k)] = haskey(simulation_settings, k) ? simulation_settings[k] : ""
     end    
 
     # check if provided paths exist
     for (param, path) in inputs
-        if (path != "") && !isfile(path)
+        if (path != "") && !(isfile(path) || isdir(path))
```

Comment:
Is `path` the name of the directory or of the file?

Reply:
In case of `:fieldgen_config, :drift_vel, :impurity_profile` it's a file, in case of `:crystal_metadata_path` it's a directory